src/gie.c

```diff
@@ -250,6 +247,13 @@ int main (int argc, char **argv) {
         return 0;
     }
 
+
+    if (opt_given (o, "version")) {
+        fprintf (stdout, "%s: %s\n", o->progname, pj_get_release ());
```

Comment:
It would be nice to use `proj_info().release` or `proj_info().version` here.

Reply:
(ibid.)